demo/core_13_$.utils.code.js

```diff
@@ -244,8 +244,8 @@ $.utils.code.rewriteForEval.unittest = function() {
       actual = SyntaxError;
     }
     if (actual !== cases[key][0]) {
-      throw Error('Eval Expression\n' +
-                  'Expected: ' + cases[key][0] + ' Actual: ' + actual);
+      throw new Error('Eval Expression\n' +
+                      'Expected: ' + cases[key][0] + ' Actual: ' + actual);
```

Comment:
Comment: the style guide is not entirely clear about how the second line should be indented.  The most obvious interpretation is that the `'` should be aligned underneath the `w` of `throw`.  I personally prefer how you have it here.

Reply:
The old style guide offered an explicit choice:
https://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting